ddtrace/commands/ddtrace_run.py

```diff
@@ -1,11 +1,22 @@
 #!/usr/bin/env python
+import argparse
 from distutils import spawn
 import os
 import sys
 import logging
 
-debug = os.environ.get('DATADOG_TRACE_DEBUG')
-if debug and debug.lower() == 'true':
+import ddtrace
+from ddtrace.compat import PY2
+
+
+if PY2:
+    # Python 2 does not have PermissionError but Python 3 does.
+    class PermissionError(Exception):  # noqa
+        pass
+
+
+debug = os.environ.get("DATADOG_TRACE_DEBUG")
```

Comment:
should we do `get_env("trace", "debug")`?

Reply:
should we set `os.environ["DD_TRACE_DEBUG"] = "true"` when this is enabled as well?

Reply:
Yeah we can update this with these changes